egi/api_resources/gamelisting/checkin.py

```diff
@@ -87,6 +96,14 @@ def post(self):
             abort(400, message="You must specify at least one of "
                                "web_client_url or telnet_hostname+telnet_port.")
 
+        # handle Evennia 0.6 and prior EGI keys
+        if not args.connected_account_count \
+           and args.connect_player_count:
+            args.connected_account_count = args.connected_player_count
```

Comment:
Since we are passing in all of these args as `**args` down on line 94, do we know what'll happen to our now-non-DB-backed `connect_player_count` value? Mostly just wanted to verify that we're not going to run into an error.

Reply:
Good catch, it seems worth doing anyway! There wasn't an error when I ran the server, but it's good practice.

Reply:
If there's no error, I'm A-OK with doing nothing.

Reply:
Eh, it seems better to not hand down attributes we're not expecting. Tiny tiny memory savings, maybe?

Reply:
Whoops, not ready yet, getting an error in the namespace, one moment...